src/components/Modal/Modal.vue

```diff
@@ -525,6 +527,59 @@ export default {
 				this.slideshowTimeout.clear()
 			}
 		},
+		/**
+		 * Get List of focusableelmets from content container
+		 */
+		getFocusableList() {
+			const contentContainer = this.$refs.mask.querySelector('.modal-container')
+			const focusableList = contentContainer.querySelectorAll(
+				'button, [href], input, select, textarea, [tabindex]:not([tabindex="-1"], input:not([type="hidden"])'
+			)
+			return focusableList
+
+		},
+		/**
+		 * set focus on first focusable element in content container
+		 */
+		setFocusOnFirstFocusable() {
+			const focusableList = this.getFocusableList()
+			const firstFocusable = focusableList[0]
+			this.$nextTick(function() {
+				if (firstFocusable) {
+					firstFocusable.focus()
+				}
+			})
+		},
+		/**
+		 * set focus trap
+		 *
+		 * @param e
+		 */
+		checkForTabKeyEvent(e) {
```

Comment:
did you know about the "inert" attribute ? https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert

it's not supported yet (or experimentally) by browsers but there's a polyfill for it

Reply:
@PVince81 thanks. I thought maybe [https://github.com/focus-trap/focus-trap](https://github.com/focus-trap/focus-trap) is a good solution.